src/RokuDeploy.spec.ts

```diff
@@ -699,182 +637,6 @@ describe('index', () => {
 
     });
 
-    describe('createPackage', () => {
```

Comment:
I think we should bring back most of these tests. Looks like maybe we just need to tweak some of them to test against the new rokuDeploy function names?

Reply:
Done